base/logging.go

```diff
@@ -483,6 +483,25 @@ func UpdateLogger(logFilePath string) {
 	}
 }
 
+// This provides an io.Writer interface around the base.Log API
+type LoggerWriter struct {
+	LogKey string  // The log key to log to, eg, "HTTP++"
+
+}
+
+// Write() method to satisfy the io.Writer interface
+func (lw *LoggerWriter) Write(p []byte) (n int, err error) {
+	printf(fgYellow+lw.LogKey+": "+reset+"%s", string(p))
+	return len(p), nil
```

Comment:
This doesn't provide much context for the logged request (which HTTP request it was, request #, which user, etc).  Under load, I expect it might be hard to correlate HTTP++ output with the corresponding HTTP and HTTP+ logging in the logs.  In some cases that won't matter, but in others I think it would be critical to identify which request this was the body for.